src/main/java/com/autotune/analyzer/kruizeObject/KruizeObject.java

```diff
@@ -47,6 +47,8 @@ public final class KruizeObject {
     private String clusterName;
     @SerializedName("datasource")
     private String datasource;
+    @SerializedName(KruizeConstants.JSONKeys.EXPERIMENT_TYPE) //TODO: to be used in future
+    private String experiment_type;
```

Comment:
Can you please make it camel case
private String experimentType;

Reply:
Updated to use camel case.